packages/react/package.json

```diff
@@ -56,6 +54,7 @@
         "d3-scale": "4.0.2",
         "d3-shape": "3.2.0",
         "dnd-core": "16.0.1",
+        "escape-string-regexp": "4.0.0",
```

Comment:
```suggestion
        "escape-string-regexp": "5.0.0",
```

Reply:
This is a deliberate change, not an error :P 

v5 is ESM only: https://github.com/sindresorhus/escape-string-regexp/releases/tag/v5.0.0

Reply:
It does work currently though, no?

Reply:
It works only because we are using `"moduleResolution": "node"` and this resolver doesn't properly understand the `"type": "module"` field of dependencies. As soon as we switch to something else like `node16`, we get the following error when building the declarations for the `cjs` output. 

```
src/components/partial-string-match/PartialStringMatch.tsx:1:32 - error TS1479: The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("escape-string-regexp")' call instead.
  To convert this file to an ECMAScript module, add the field `"type": "module"` to '/Users/gdostie/Developer/coveo/plasma/packages/react/package.json'.

1 import escapeStringRegexp from 'escape-string-regexp';
```

Reply:
There are 2 ways to be safe with v5:
- Stop distributing commonjs output (be an ESM only package ourselves)
- Somehow install the 2 versions (v4 and v5) and build cjs with v4 and esm with v5